ui/src/shared/components/TimeSeries.tsx

```diff
@@ -271,6 +280,12 @@ class TimeSeries extends Component<Props & WithRouterProps, State> {
 
       this.pendingReload = false
 
+      const queryText = queries[0].text
```

Comment:
What happens if there is more than one query in queries ?

Reply:
I was wondering about that. I've never seen a situation where more than one query is being sent, but is that a possibility? If so, should we cache each one separately?